Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,42 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow", :using => :git,
+    :tag => "v1.0.0", :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
+    system 'echo "\n\n\n\n\n\n\n\n\n" | ./configure'
```

Comment:
@kali I'd advise submitting a patch upstream (or at least an issue) that fixes this problem. This is particularly hostile to any package manager packaging this software as-is.